ci/tests/binary_tree.c

```diff
@@ -0,0 +1,117 @@
+/* The Computer Language Benchmarks Game
+ * https://salsa.debian.org/benchmarksgame-team/benchmarksgame/
+
+ contributed by Kevin Carson
```

Comment:
Somewhere (in this file, or a `COPYRIGHT` / `README` / etc file) we need to explicitly state what the copyright on this file is.

Reply:
Added copyright in the file directly in commit [3d1414c](https://github.com/capablevms/bdwgc/pull/13/commits/3d1414c66d1106e3a54a10e2c4e3abcc8d426fbe).
2-clause BSD license , so should be OK to put up here ?

Reply:
Works for me!